packages/client/src/components/UploadForm.tsx

```diff
@@ -41,6 +45,10 @@ export const UploadForm = ({ auth }: { auth: AuthState }) => {
 			`/signedUrl?${urlParams.toString()}`,
```

Comment:
think we need to change this to `/api/signedUrl` given the changes to `authFetch`